### PR TITLE
WV-490 parse politician name and state, render on before load

### PR DESCRIPTION
**Pull request by omgitsmiles**
### What github.com/wevote/WebApp/issues does this fix?

Google  “de-lists” the politician page when there is a delay in retrieving politician information. The information does seem to eventually load, but too late for Google. I parsed the politician name and state from URL to render on suspense before api loads. 

### Changes included this pull request?

- extractPoliticianDetailsFromUrl function pulling politician name and state from URL
- adding a suspense with politician details on load

To Dos:

- [ ] Code cleanup / ES Lint

### CODE REVIEW

File: src/js/compileDate.js.backup [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = ['10/9/2024, 5:54:14 PM'];
```
Comment (DaleMcGrew) [inline src/js/compileDate.js.backup]: Please add this compileDate.js.backup file to .gitignore so it isn't checked in.

File: src/js/common/pages/Politician/PoliticianDetailsPage.jsx [hunk 2]
```diff
@@ -901,6 +942,14 @@ class PoliticianDetailsPage extends Component {
           <meta name="description" content={politicianDescriptionLimited} />
         </Helmet>
         <PageWrapper>
+        <Suspense fallback={(
```
Comment (DaleMcGrew) [inline src/js/common/pages/Politician/PoliticianDetailsPage.jsx]: Hi @omgitsmiles, we don't want to add a different place to display this information, we want the politicianNameParsedFromURLBeforeLoad and politicianStateParsedFromURLBeforeLoad placed into the components where the name and state currently appear once the data loads.

**Review by DaleMcGrew — CHANGES_REQUESTED** [review]
This is a good start @omgitsmiles, thank you. Comments included.

File: src/js/common/pages/Politician/PoliticianDetailsPage.jsx (hunk 2 above)
Comment (omgitsmiles) [reply src/js/common/pages/Politician/PoliticianDetailsPage.jsx]: Thanks for the clarification. Will update.

**Comment by DaleMcGrew** [comment]
Thank you @omgitsmiles! 👍

**Pull request merged**